bears/go/GoTOMLBear.py

```diff
@@ -0,0 +1,22 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a file.
+    """
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {GoRequirement('github.com/BurntSushi/toml/cmd/tomlv',
+                                  version='0.1.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ()
```

Comment:
CMIIW (I am still learning about writing bears) but is this really correct that the linter doesnt need filename args to be returned?